docs/iris/src/developers_guide/pulls.rst

```diff
@@ -62,20 +63,20 @@ The Iris Check List
 
  * ``make doctest``, ``make extest``  in ``./docs/iris``.
```

Comment:
`extest` is a target in the documentation Makefile that runs tests on the example code (code that produces gallery examples).